Userland/Libraries/LibGfx/ImageFormats/TIFFLoader.cpp

```diff
@@ -360,8 +360,10 @@ class TIFFLoadingContext {
 
         auto const number_of_field = TRY(read_value<u16>());
 
-        for (u16 i = 0; i < number_of_field; ++i)
-            TRY(read_tag());
+        for (u16 i = 0; i < number_of_field; ++i) {
+            if (auto maybe_error = read_tag(); maybe_error.is_error() && TIFF_DEBUG)
```

Comment:
Maybe add a comment saying that we intentionally ignore tags that fail to decode.

Reply:
Do we really need that? When I read code, I consider the explicit `Error` management dance clear enough to know that the author intentionally did something special with the error. And in the worst case scenario, the commit message also supports the fact that this is very intentional.

Reply:
imho yes: There's no test protecting this from regressing, and `git blame` output becomes replaced through unrelated changes really quickly – try to look through `blame` output on an older file. It's usually 3-5 hops, often across a few renames, until you find the original commit message.

Reply:
Then we should write a test for protecting it from regressing, not write a comment!
And thanks for saying that, I discovered a bug by adding this test!